src/binder/bind/bind_copy.cpp

```diff
@@ -213,16 +204,29 @@ expression_vector Binder::bindExpectedNodeFileColumns(
             if (skipPropertyInFile(*property)) {
                 continue;
             }
-            readerConfig.columnNames.push_back(property->getName());
-            readerConfig.columnTypes.push_back(property->getDataType()->copy());
-            columns.push_back(createVariable(property->getName(), *property->getDataType()));
+            expectedColumnNames.push_back(property->getName());
+            expectedColumnTypes.push_back(property->getDataType()->copy());
```

Comment:
Inline this entire thing into:

```
if (!skipPropertyInFile(*property)) {
```

Reply:
Specifically I mean:

```
if (!skipPropertyInFile(*property)) {
    expectedColumnNames.push_back(property->getName());
    expectedColumnTypes.push_back(property->getDataType()->copy());
}
```

Reply:
Is there a specific advantage to the inlined version? Normally I avoid use `!condition`.

Reply:
I avoid continue as a rule. Personally I find it harder to read. I dont have anything against negated conditions personally.